src/Raven.Studio/wwwroot/App/views/database/query/query.html

```diff
@@ -216,36 +222,42 @@ <h2 class="on-base-background">
                 <div class="panel-body absolute-fill js-time-series-graph-container" data-bind="if: showTimeSeriesGraph(), visible: showTimeSeriesGraph()">
                     <div class="absolute-fill flex-vertical" data-bind="compose: $root.currentTab"></div>
                 </div>
+                <div class="panel-body absolute-fill js-time-series-graph-container" data-bind="if: showMapView(), visible: showMapView()">
+                    <div class="absolute-fill flex-vertical" data-bind="compose: $root.currentTab"></div>
+                </div>
             </div>
         </div>
         <div class="flex-window-head">
-            <button type="button" class="btn btn-sm" 
+            <button type="button" class="btn btn-sm"
                     data-bind="css: { active: currentTab() === 'results'}, click: goToResultsTab">
                 Results &nbsp; <span class="label" data-bind="text: totalResultsForUi().toLocaleString() + (hasMoreUnboundedResults() ? '+' : ''), css: { 'label-primary': totalResultsForUi(), 'label-default': !totalResultsForUi() }"></span>
             </button>
+            <button type="button" class="btn btn-sm" data-bind="click: $root.goToSpatialMapTab, css: { active: showMapView }, visible: viewOnMapClicked">
+                <span>Spatial Map</span> &nbsp; <span class="label" data-bind="text: totalResultsForUi().toLocaleString() + (hasMoreUnboundedResults() ? '+' : ''), css: { 'label-primary': totalResultsForUi(), 'label-default': !totalResultsForUi() }"></span>
```

Comment:
this is probably not true since we don't load all records to memory, right?

Reply:
So currently the number in the label is the same as on the Results tab.
As discussed, we will change this implementation in https://issues.hibernatingrhinos.com/issue/RavenDB-16009

Reply:
great idea